lib/actions/local/configure.coffee

```diff
@@ -95,100 +67,65 @@ inquirerOptions = (data) -> [
 	}
 ]
 
+joinPath = (path) -> '/' + path.join('/')
+
+load = (target, type) ->
+	{ Reconfix } = require('reconfix-preview')
+	imagefs = require('resin-image-fs')
+
+	reconfix = new Reconfix({
+		read: (partition, path) ->
+			imagefs.readFile
+				image: target
+				partition: partition
+				path: joinPath(path)
+			.catch (err) ->
+				if err?.code == 'NOENT'
```

Comment:
Shouldn't this be `ENOENT`?

Reply:
Why is it necessary to check if `err` exists? It should always be present on a catch block, right?

Reply:
`NOENT` is what the error type that `resin-image-fs` emits. It would be best to standardize (or document) the error interface of `resin-image-fs`.